rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -58,7 +58,7 @@
   private static Log log = LogFactory.getLog(BookKeeperServer.class.getName());
   private static BookKeeperMetrics bookKeeperMetrics;
 
-  protected BookKeeperServer()
+  public BookKeeperServer()
```

Comment:
are we using this constructor outside this bookkeeper package ? If not we should not be making this public. Method without any modifier is accessible within the same package. So we should just use no modifier if its only used in the same package 

This applies to all other places where we are using public modifier

Reply:
It's used in tests like `TestCachingInputStream` in `rubix-tests` for starting the BookKeeperServer, now that I've removed the static modifiers.

https://github.com/qubole/rubix/blob/af9089df037e572d3265366e266589756abc8859/rubix-tests/src/test/java/com/qubole/rubix/tests/TestCachingInputStream.java#L106-L114

Reply:
ok .. makes sense